src/main/java/screens/Factory.java

```diff
@@ -0,0 +1,69 @@
+package screens;
+
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * Factory for JavaFX constructors.
+ */
+public class Factory {
```

Comment:
Factory is a rather ambiguous name, could you give this class a more descriptive name?